gateway/api/views.py

```diff
@@ -412,13 +413,8 @@ def get_jobs(
                 jobs = Job.objects.filter(program=program)
             else:
                 jobs = Job.objects.filter(program=program, author=request.user)
-            return Response(
-                list(
-                    jobs.values(
-                        "status", "result", "id", "created", "version", "arguments"
-                    )
-                )
-            )
+            serializer = JobSerializer(jobs, many=True)
```

Comment:
can we create a serializer that doesn't return logs and results? Thinking in return the minimum information as possible to reduce the size of the query